server/services/bookingService.test.ts

```diff
@@ -72,6 +80,26 @@ describe('BookingService', () => {
     })
   })
 
+  describe('getKeyworkers', () => {
+    it('should return the keyworker data needed', async () => {
+      const keyWorkers = referenceDataFactory.buildList(2)
+
+      referenceDataClient.getReferenceData.mockImplementation(category => {
```

Comment:
We could just `.mockResolvedValue(keyWorkers)` here? Or am I missing something?

Reply:
Yeah, probs a bit overengineered!